### DISCUSSION TITLE
hackintosh/updates/2023/04/03/acidanthera-april

### DISCUSSION THREAD

**Discussion by giscus** [dortania/dortania.github.io#43] [Comments]
# hackintosh/updates/2023/04/03/acidanthera-april

Acidanthera Updates: A message from PMheart Changelogs Acidanthera Updates: A message from PMheart Embracing a warm spring, we published our 2023.04 releases. OpenCore 0.9.1 brings better support for legacy graphic cards together with bug fixes. To begin with, @corpnewt fixed a minor printing bug in ocvalidate which led to the failure of message printing. @mikebeaton made VS Code debugging easier by providing sample config for gdb. As well, he implemented GopBurstMode quirk for faster GOP operation on older firmware; now SysReport will also include GOP memory caching reports. Thanks to @vit9696’s discovery, we also integrated the new patch for ThirdPartyDrives quirk on macOS 13.3 and above. Last but not least, @Andrey1970AppleLife has always continuously updated builtin firmware versions. macOS Ventura introduced a new check for Continuity Camera which is only available on Kaby Lake and above. FeatureUnlock 1.1.4 patches this restriction to make it possible everywhere thanks to @khronokernel. Users’ consistent maintenance to AppleALC database has been integrated into AppleALC 1.8.1 as well. We would like to thank @lgh07711, @DalianSky, @Pinokyo-H, @wern-apfel, and @imoize, for their contibutions to the configurations of ALC293, Alder Lake PCH-P High Definition Audio Controller (0x51C8), ALC269, ALC623, ALC295, and ALC255 respectively. To investigate NSS issues, @lvs1974 updated AirportBrcmFixup to 2.1.7 to dump debugging logs from the methods AirPort_BrcmNIC::setTX_NSS, AirPort_BrcmNIC::getTX_NSS, and AirPort_BrcmNIC::getNSS. As of RestrictEvents 1.1.0, it is possible to disable hw.optional.f16c on macOS 13.3 and above. This is particularly useful on Ivy Bridge CPUs where AVX 2.0 instructions are not present, resulting in CoreGraphics.framework crash. We appreciate @AsdMonio who fixed board-id length calculation for legacy Mac models where it is shorter. This is now integrated into BrcmPatchRAM 2.6.5. In BrightnessKeys 1.0.3, @ChefKissInc added support for certain laptops including AMD ones where kIOACPILegacyPanel is used. Thanks to @1Revenger1, one of the primary authors of our VoodooPS2 project, version 2.3.5 removed actAsTrackpad and related logic; and on Elan Touchpads, Trackpoint logic is now powered by VoodooInput. Enjoy, — PMheart Changelogs OpenCorePkg 0.9.1 Fixed long comment printing for ACPI patches, thx @corpnewt Added sample config for VS Code source level debugging with gdb Updated builtin firmware versions for SMBIOS and the rest Added GOP memory caching report to SysReport Implemented GopBurstMode quirk for faster GOP operation on older firmware Fixed ThirdPartyDrives quirk on macOS 13.3 and above FeatureUnlock 1.6.4 Added Continuity Camera unlocking for pre-Kaby Lake CPUIDs AppleALC 1.8.1 Added ALC293 layout-id 31 for Hasee Z7-CT7NA by lgh07711 Added Alder Lake PCH-P High Definition Audio Controller (0x51C8) by DalianSky Added ALC269 layout-id 111 for minisforum NAG6 by DalianSky Added ALC623 layout-id 13 for Lenovo ThinkCentre M720e with internal speaker by Pinokyo-H Added ALC295 layout-id 11 for ZenBook UX581 by wern-apfel Added ALC255 layout-id 37 for Acer Nitro 5 AN515-52-73Y8 by imoize AirportBrcmFixup 2.1.7 Override methods AirPort_BrcmNIC::setTX_NSS, AirPort_BrcmNIC::getTX_NSS and AirPort_BrcmNIC::getNSS to investigate NSS issues. Original method is called, and debug version of kext (with boot-arg -brcmfxdbg) prints info into log RestrictEvents 1.1.0 Added hw.optional.f16c disabling for macOS 13.3+ Resolves CoreGraphics.framework invoking AVX2.0 code paths on Ivy Bridge CPUs Configurable via revpatch’s f16c argument BrcmPatchRAM 2.6.5 Fixed legacy Mac compatibility (thx @AsdMonio) BrightnessKeys 1.0.3 Added legacy panel support in ACPI (thx @ChefKissInc) VoodooPS2 2.3.5 Removed actAsTrackpad and related logic Fix Trackpoints connected to Elan Touchpads Use VoodooInput Trackpoint logic for Elan Touchpads

https://dortania.github.io/hackintosh/updates/2023/04/03/acidanthera-april.html

**Comment by LanceScs** [comment]
OCPL 6.4 is great and I used it to install Ventura on MBP 15", 10,1 mid 2012.(4c I7, 16GB)
Everything went very smooth, thank you for that but it doesn't support Intel HD Graphics 4000 and NVIDIA GeForce GT 650M . The Post Install did not find the right graphic cards. Could you pls advise whether there is a workaround for these graphic cards in existence?

**Reply by RickiThaDragon** [reply]
I assume You mean You are using OCLP, and as for Grpahics Accelleration for HD 4000 You should be able to use post install patches which Your IvyBridge & Nvidia (Kepler) are supported bro as far as i can tell. I do think though that You will have better experience overall with monterey or big sur on that hardware You speak of.

**Comment by H3CKD4SY5T3M** [comment]
Hello,
OCPL 0.6.4 and 0.6.5 : installation on MBP 17" 6,1 mid 2010 (Intel i7, 8GB) success ! Thanks.
Several reboots and "post install root patch" updates needed.
Bugs that still need to be fixed : 
1- Airdrop doesn't work (find no iOS devices nor MBP)
2- iMovie 10.3.5 crashes once started
3- Final cut Pro 10.6.5 crashes once started

Thanx

**Reply by H3CKD4SY5T3M** [reply]
I forgot : apple app Plans also doesn't work.
So, it needs also a fix.

**Comment by LanceScs** [comment]
Well, așa matter of fact, I am running BS on this old computer and it runs fine, arguable of course.
Thanks for your advice.

Reds, etc

> On May 6, 2023, at 4:45 AM, RickyThaDragon ***@***.***> wrote:
> 
> 
> I assume You mean You are using OCLP, and as for Grpahics Accelleration for HD 4000 You should be able to use post install patches which Your IvyBridge & Nvidia (Kepler) are supported bro as far as i can tell. I do think though that You will have better experience overall with monterey or big sur on that hardware You speak of.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/dortania/dortania.github.io/discussions/43#discussioncomment-5821984>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR7QQ3S26MGJFLJF7HNFGZLXEW3K3ANCNFSM6AAAAAAWRVZEJM>.
> You are receiving this because you commented.
> 


 


             															



Andre van der Lans
***@***.***

**Comment by RickiThaDragon** [comment]
No problem.I was just trying to be helpful.Theres no need to be uptight. Just trying to help
